<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="1">
    <widget class="QPushButton" name="deleteTrace">
     <property name="text">
      <string>Delete Trace</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QTableWidget" name="traceTable">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Trace ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Layer</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>X</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Y</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Color</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Transparency</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="addTrace">
     <property name="text">
      <string>Add Trace</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QToolButton" name="browseButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Plot Settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="5">
          <widget class="QSpinBox" name="alphaBox">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="2" column="3" colspan="2">
          <widget class="QSlider" name="alpha">
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="invertedControls">
            <bool>false</bool>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Outline Width</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QgsFieldExpressionWidget" name="expField1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="3" column="3" colspan="2">
          <widget class="QComboBox" name="barCombo">
           <item>
            <property name="text">
             <string>group</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>stack</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QDoubleSpinBox" name="widthBox">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Fill color</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Bar Mode</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Layer</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QgsMapLayerComboBox" name="LayerCombo"/>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Transparency</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>X</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Y</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Outline color</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QgsFieldExpressionWidget" name="expField2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QgsColorButton" name="mColorButton"/>
         </item>
         <item row="1" column="3">
          <widget class="QgsColorButton" name="mColorButton_2"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Layout Customization (optional)</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="3">
          <widget class="QCheckBox" name="yAxisCheck">
           <property name="text">
            <string>Y Axis Label</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QCheckBox" name="xAxisCheck">
           <property name="text">
            <string>X Axis Label</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="legendCheck">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Show Legend</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="titleCheck">
           <property name="text">
            <string>Plot Title</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="pltTitle">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string>Title</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Bar orientation</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QComboBox" name="orientationCombo">
           <item>
            <property name="text">
             <string>Vertical</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Horizontal</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLineEdit" name="xAxisText">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLineEdit" name="yAxisText">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="filePath">
     <property name="text">
      <string>Temporary file</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsColorButton</class>
   <extends>QPushButton</extends>
   <header>qgscolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QgsFieldExpressionWidget</class>
   <extends>QWidget</extends>
   <header>qgsfieldexpressionwidget.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>613</x>
     <y>491</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LayerCombo</sender>
   <signal>layerChanged(QgsMapLayer*)</signal>
   <receiver>expField1</receiver>
   <slot>setLayer(QgsMapLayer*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LayerCombo</sender>
   <signal>layerChanged(QgsMapLayer*)</signal>
   <receiver>expField2</receiver>
   <slot>setLayer(QgsMapLayer*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>alpha</sender>
   <signal>valueChanged(int)</signal>
   <receiver>alphaBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>628</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>706</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>alphaBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>alpha</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>706</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>628</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>titleCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>pltTitle</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xAxisCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>xAxisText</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yAxisCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>yAxisText</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
